init.go

```diff
@@ -1,11 +1,12 @@
 package router
 
 import (
-	"github.com/gopherjs/gopherjs/js"
+	"github.com/gopherjs/gopherwasm/js"
 )
 
 func init() {
-	js.Global.Set("onpopstate", func(e *js.Object) {
-		refreshRoutes()
-	})
+	js.Global().Set("onpopstate", js.FuncOf(func(this js.Value, args []js.Value) interface{} {
+		go refreshRoutes()
```

Comment:
Out of curiosity, any reason why this is called in a goroutine? re-rendering is a synchronous process anyway and this is a callback so everything is already asynchronous.